ui-lib/src/main/res/ui/warning/values/strings.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="not_enough_space_title">Not enough space!</string>
+    <string name="not_enough_space_description">You need approximately %1$s gig of space while synchronizing the Zcash blockchain, but only 300 megs once done. Syncing will stay paused until more space is available.</string>
```

Comment:
Can you update the strings to use xliff?  Some of our other strings have examples.  It'll help once we need to hire localization help.